front-end/src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const APP = 'polkassembly';
+export const NETWORK = 'kasuma';
+export const WS_PROVIDER = 'wss://kusama-rpc.polkadot.io'; // 'ws://127.0.0.1:9944';
```

Comment:
should we keep these values in enviroment variables. Or here is fine @Tbaut . If we have them here we need to change them each time we are developing on local. At least for WS_PROVIDER